crates/bevy_ecs/src/system/commands.rs

```diff
@@ -189,38 +190,79 @@ pub struct Commands {
 }
 
 impl Commands {
-    pub fn spawn(&mut self, components: impl DynamicBundle + Send + Sync + 'static) -> &mut Self {
+    /// Creates a new entity and calls `insert` with the it and `bundle`.
+    ///
+    /// Note that `bundle` is a bundle. If you would like to spawn an entity with a single component, consider wrapping the component in a tuple (which `Bundle` is implemented for).
+    ///
+    /// See [`Commands::set_current_entity`], [`Commands::insert`].
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use bevy_ecs::prelude::*;
+    ///
+    /// struct Component1;
+    /// struct Component2;
+    ///
+    /// #[derive(Bundle)]
+    /// struct ExampleBundle {
+    ///     a: Component1,
+    ///     b: Component2,
+    /// }
+    ///
+    /// fn example_system(mut commands: Commands) {
+    ///     // Create a new entity with a component bundle.
+    ///     commands.spawn(ExampleBundle {
+    ///         a: Component1,
+    ///         b: Component2,
+    ///     });
+    ///
+    ///     // Create new entities with a single component each.
+    ///     commands.spawn((Component1,));
+    ///     commands.spawn((Component2,));
```

Comment:
Lets show spawning with multiple components too instead of duplicating a "single component" example

```rust
// Create a new entity with a single component
commands.spawn((Component1,));
// Create a new entity with two components
commands.spawn((Component1, Component2,));
```